.github/workflows/run-make-all.yml

```diff
@@ -31,16 +31,23 @@ jobs:
         if: ${{ github.ref == 'refs/heads/master' }}
         run: |
           RELEASE_TAG=$(git describe --tags)
-          mkdir -p artifacts
           docker run --rm catalalang/catala-build:${IMAGE_TAG} sh -uexc '
```

Comment:
Maybe you could make that line more robust see https://github.com/CatalaLang/catala/pull/423#issuecomment-1458634308